queries/nix/locals.scm

```diff
@@ -0,0 +1,15 @@
+; let bindings
+(let (bind . (attrpath) @definition.var)) @scope
+
+; rec attrsets
+(rec_attrset (bind . (attrpath) @definition.field)) @scope
+
+; functions and parameters
+(function . [
+    (identifier) @definition.parameter
+    (formals (formal . (identifier) @definition.parameter))
+]) @scope
+((formals) "@" (identifier) @definition.parameter) ; I couldn't get this to work properly inside the (function)
```

Comment:
We don't have (cheep) semantic highlighting yet

Reply:
What do you mean? I've been using `nvim-treesitter-refactor` for testing this. The issue I was encountering was that for some reason if I combined the two `(formals)` instances then the identifiers wouldn't get `@definition.parameter` if there was an `@ identifier`.

Reply:
ah, sorry for my confusion.